From 6b5978dcf1f7ac3ecc2f22df06f7000f360e2066 Mon Sep 17 00:00:00 2001
From: Richard Hughes <richard@hughsie.com>
Date: Mon, 17 Aug 2009 08:15:32 +0000
Subject: Do not reset lastDeviceEventTime when we do dixSaveScreens

When we turn off DPMS with DPMSModeOff and do dixSaveScreens, don't reset the
event time else session clients using IDLETIME will be reset.

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
---
diff --git a/dix/window.c b/dix/window.c
index 32e26d9..1a645f6 100644
--- a/dix/window.c
+++ b/dix/window.c
@@ -3128,8 +3128,6 @@ dixSaveScreens(ClientPtr client, int on, int mode)
 
     if (on == SCREEN_SAVER_FORCER)
     {
-	UpdateCurrentTimeIf();
-	lastDeviceEventTime = currentTime;
 	if (mode == ScreenSaverReset)
 	    what = SCREEN_SAVER_OFF;
 	else
--
cgit v0.8.2
